app/code/Magento/Catalog/Api/CategoryListDeleteBySkuInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Catalog\Api;
+
+/**
+ * @api
+ * @since 100.0.2
+ */
+interface CategoryListDeleteBySkuInterface
+{
+    /**
+     * Delete by skus list
+     *
+     * @param int   $categoryId
+     * @param array $productSkuList
+     * @return bool
+     *
+     * @throws \Magento\Framework\Exception\CouldNotSaveException
```

Comment:
There should be also added NoSuchEntityException

Reply:
Hello @ihor-sviziev, may be add and  Magento\Framework\Exception\InputException ?